daml/Marketplace/Custody/Model.daml

```diff
@@ -1,4 +1,14 @@
-module Marketplace.Custody.Model where
+module Marketplace.Custody.Model (
+    OpenAccountRequest(..)
+  , OpenAllocationAccountRequest(..)
+  , CloseAccountRequest(..)
+  , CloseAllocationAccountRequest(..)
+  , CreditAccountRequest(..)
+  , DebitAccountRequest(..)
+  , TransferDepositRequest(..)
+  , LifecycleRequest(..)
+  , Process(..)
```

Comment:
```suggestion
```

Also please remove comment line110.

Reply:
Service.daml, line 140:
```
Custody.Process
```
I cannot remove that.

Reply:
Ah, ok. I remember now we refactored from a single file to multiple `Model/Service` files, so this no longer applies. Sorry, my bad.